server/api.js

```diff
@@ -56,9 +56,13 @@ module.exports = function (app) {
         mutations.getGrantTokenForUser(req.session.user.trayId)
             .then(({payload}) => {
                 const authorizationCode = payload.data.generateAuthorizationCode.authorizationCode;
+                const popupUrl = req.body.solutionInstanceId && req.body.externalAuthId ?
+                    `${createAuthPath}/${req.body.solutionInstanceId}/${req.body.externalAuthId}?code=${authorizationCode}` :
+                    `${createAuthPath}?code=${authorizationCode}`;
+
                 res.status(200).send({
                     data: {
-                        popupUrl: `${createAuthPath}/${req.body.solutionInstanceId}/${req.body.externalAuthId}?code=${authorizationCode}`
+                        popupUrl 
```

Comment:
do we need the space after?

Reply:
Hmm nope, not necessary. I've been wanting to lint the whole repo for a while, so I'm going to leave it like this and do that in a future PR.